pepys_import/file/replay_comment_importer.py

```diff
@@ -108,8 +108,10 @@ def load_this_file(self, data_store, path, file_object, datafile):
                     self.name, "timestamp", timestamp, "n/a"
                 )
 
-                message = message_token.text
-                message_token.record(self.name, "message", message, "n/a")
+                message = " ".join([t.text for t in message_tokens])
+                combine_tokens(*message_tokens).record(
```

Comment:
Aah, this was the Python that threw me, the use of the `*` operator.

Reply:
We could add a comment to explain it if you want? It basically turns a list into a set of arguments to a function. So here if there are five tokens in `message_tokens` then it passes those as five arguments to `combine_tokens`.

Reply:
No, it's ok @robintw .  We have to assume a basic level of Python skills.